modules/custom/az_core/config/install/core.entity_view_mode.taxonomy_term.az_badge_alt.yml

```diff
@@ -0,0 +1,9 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - taxonomy
+id: taxonomy_term.az_badge_alt
+label: Alternate Badge
```

Comment:
Is there a reason this is named "Alternate Badge" rather than "Linked Badge"? Are there other functionality differences that I'm missing?

Reply:
No good reason other than trying to keep the name flexible.